lighthouse-core/audits/without-javascript.js

```diff
@@ -28,6 +28,7 @@ class WithoutJavaScript extends Audit {
       category: 'JavaScript',
       name: 'without-javascript',
       description: 'Page contains some content when its scripts are not available',
+      helpText: 'Your app should display some content when JavaScript is disabled, even if it\'s just a warning to the user that JavaScript is required to use the app. <a href="https://developers.google.com/web/tools/lighthouse/audits/no-js" target="lighthouse" rel="noreferrer noopener">Learn more</a>.',
```

Comment:
That would make it redundant with the section's title and description. Similar to how "load time is a key factor in how users perceive your app" was redundant in the perf audit help texts that I wrote.

![pe](https://cloud.githubusercontent.com/assets/4713486/20537405/e4e164ea-b0a1-11e6-91c4-e1570b31fe74.png)